017/master

BenGitsCode opened the pull request:
- Refresh for 017
- Initial update carried over from unmerged https://github.com/ga-wdi-boston/js-objects-this/pull/34

BenGitsCode committed:
Update from talk-template

-  Refresh for cohort LM-01.
-  .gitignore update.

BenGitsCode committed:
Update from node-template

-  Refresh for cohort LM-01.

BenGitsCode committed:
Fix grammatical error in README

-  Address and close #29
-  That wasn't a sentence. Now it is.

BenGitsCode committed:
Add additional resources for understanding this

-  Address and close #23
-  Added a javascript is sexy article and a helpful blogpost.

BenGitsCode committed:
Space after function keywords in README

-  Address and close #30

BenGitsCode committed:
Make-standard

-  Ran make-standard task and added modified files.

BenGitsCode committed:
Add .eslintrc.json spec file missed in node update

-  Address comments on #34

BenGitsCode committed:
Rename package.json title to correct repo name

-  Named package.json title js-objects-this.
-  Was erroneously updated to node-template.
-  Close #35

BenGitsCode committed:
Correct spaces after function in code snippets

-  Close #30

bernardlee reviewed (changes requested):
- Add bin/ and example files from node-template back in so that `grunt server` doesn't cause an error.  Alternatively, remove the `grunt server` alias.
- Remove grunt-contrib-jshint and grunt-jscs from package.json.

BenGitsCode committed:
Add bin/ and example files from node-template back

-  grunt server will fail without the example files.
-  Address comments #42

BenGitsCode committed:
Remove extraneous dependencies from package.json

-  Remove grunt-contrib-jshint and grunt-jscs from package.json.
-  Address comments in #42

BenGitsCode commented:
@bernardlee addressed in 	80e0d7f 06a9162
